code/obj/blob.dm

```diff
@@ -118,9 +120,17 @@
 		var/list/pa = params2list(params)
 		if ("right" in pa)
 			right_click_action()
+		else if (src.type == /obj/blob && !(("alt" in pa) || ("shift" in pa) || ("ctrl" in pa)))
+			var/mob/living/intangible/blob_overmind/user = usr
+			user.closeContextActions()
+			user.showContextActions(user.tile_upgrade_contexts, src, user.contextLayout)
```

Comment:
This seems to result in the options being a bit offset from the tile, maybe due to the circle layout not accounting for 32x32 sprites?

Reply:
Yeah not sure really, code is at /datum/contextLayout/experimentalcircle/showButtons(). Seems either intentional, or something wrong with math

![image](https://github.com/goonstation/goonstation/assets/53062374/e2c961d7-a7e7-4348-bef3-99f1133abd35)